apps/aecore/src/aec_peer_connection.erl

```diff
@@ -71,6 +73,10 @@ accept(TcpSock, Options) ->
             Res
     end.
 
+start_link([{ext_sync_port, Port}], Opts) ->
```

Comment:
Ah, it is a `simple_one_for_one` - I had some trouble seeing how this function was called with 2 arguments from start_child...